servers/wanaku-server-quarkus/src/test/java/org/wanaku/server/quarkus/support/TestIndexHelper.java

```diff
@@ -18,7 +19,7 @@ public static File createToolsIndex() throws IOException {
 
         indexFile.deleteOnExit();
 
-        if (indexFile.exists()) {
+        if (indexFile.exists() && !Files.readString(indexFile.toPath()).equals("[]")) {
```

Comment:
What's exactly the check `!Files.readString(indexFile.toPath()).equals("[]")` testing? Is it verifying if it's using MongoDB? If yes, maybe it would be good to move this to a separate method (`isMongoDB`) so it's easier to understand.

Reply:
integration tests are executed only with Files at the moment, I modified how the file is created via https://github.com/wanaku-ai/wanaku/pull/178/files#diff-fe8e9904e7d28bb1adf5f14547249fcdcb078d8edfa314504e7dbcf6598c1c24R26 and the tests need to be aligned, but I'll refactor this part